server/db/models/index.js

```diff
@@ -19,12 +19,12 @@ const ProductOrder = require('./productorder')
 Order.belongsTo(User)
 User.hasMany(Order)
 
-Order.hasMany(ProductOrder)
-ProductOrder.belongsTo(Order)
-
 Order.belongsToMany(Product, {through: ProductOrder})
 Product.belongsToMany(Order, {through: ProductOrder})
 
+// Order.hasMany(ProductOrder)
+// ProductOrder.belongsTo(Order)
+
```

Comment:
don't think we need this association

Reply:
that's what I had initially. Since we have the 2 lines above, we don't need the bellow 2 associations: 
Order.hasMany(ProductOrder)
ProductOrder.belongsTo(Order)

I checked all the associations therefore they r not needed :)